.github/CODEOWNERS

```diff
@@ -81,7 +81,7 @@ BUILD @christophermaier
 /src/rust/rust-proto/ @grapl-security/wg-data-infra
 /src/rust/rust-toolchain.toml @inickles-grapl
 /src/rust/sqs-executor/ @inickles-grapl @grapl-security/wg-data-infra
-/src/rust/sysmon/ @inickles-grapl
+/src/rust/sysmon_parser/ @inickles-grapl
```

Comment:
sysmon-parser pls

Reply:
I know that'd match other crates in this repo, but there's this https://rust-lang.github.io/rfcs/0940-hyphens-considered-harmful.html and the idea that we're moving this off this repo next anyway.

Thoughts?

Reply:
hyphens own

Reply:
I'll use dashes for now, we can consider switching if/when we do move.